src/aiida/tools/pytest_fixtures/daemon.py

```diff
@@ -116,7 +116,7 @@ def test(submit_and_await):
     from aiida.engine import ProcessState
 
     def factory(
-        submittable: type[Process] | ProcessBuilder | ProcessNode | t.Any,
+        submittable: type[Process] | ProcessBuilder | ProcessNode,
```

Comment:
@danielhollas Let's continue the discussion here. I agree that the `t.Any` is not needed but `type[Process]` is the correct change.

Reply:
Copying over your comment from the other PR. I'll need to think it through before I reply,

> Process -> type[Process] should be a correct change, in this function, it expect both workchain and calcjob which are subclass of Process. If it is just Process, then the issubclass(submittable, Process) part is useless because and shortcut the condition.
We can have a zoom chat if you like, can be easier.
> I agree that the t.Any is not necessary since we won't expect the function is used besides those three types. But after change to type[Process] which I think it is correct, there is not need to add ignore anywhere to make mypy work.

I will need to double check, but can you explain what do you think is exact different between types `type[Process]` and `Process`?

Reply:
Uh, yeah I think you're right, sorry. I think I was looking at wrong lines, shouldn't have replied in a rush.

I'll double check tomorrow.